DFC.Digital/DFC.Digital.Web.Sitefinity/ResourcePackages/gds_service_toolkit/MVC/Views/Layouts/service_default.cshtml

```diff
@@ -102,8 +103,10 @@
 
     @if (Html.IsDebugSymbol())
     {
+        @Html.Script(@Html.GetLocationAssetFileAndVersion("gds_service_toolkit/assets/src/frontend/js/cookie-banner.js").Replace("assets/dist/", string.Empty), "bottom")
```

Comment:
Take this line out, we should not reference each js or css file on the page, they should be added to appropriate bundle.